test/test.js

```diff
@@ -50,6 +50,7 @@ describe('Basic functionality', function() {
       .get('/')
       .type('text/plain')
       .expect('Access-Control-Allow-Origin', '*')
+      .expect('Access-Control-Max-Age', '0')
```

Comment:
The header should not be added by default. The user agent (browser) will assume `0` if not set.